fe/fe-core/src/main/java/com/starrocks/common/Config.java

```diff
@@ -448,8 +448,11 @@ public class Config extends ConfigBase {
      * if set to true, fe will ignore those id
      * or fe will exit
      */
-    @ConfField(mutable = true)
-    public static boolean ignore_unknown_log_id = false;
+    @ConfField(mutable = true, aliases = {"ignore_unknown_log_id"})
+    public static boolean ignore_unknown_operation_type = false;
+
+    @ConfField
+    public static boolean ignore_unknown_subtype = false;
```

Comment:
```suggestion
    public static boolean persist_ignore_unknown_subtype = false;
```

Reply:
public static boolean persist_ignore_unknown_operation_type = false;?

Reply:
done, change to metadata_ignore_unknown_subtype